javascript-loops-practice/find-index.js

```diff
@@ -0,0 +1,13 @@
+/* exported findIndex */
+function findIndex(array, value) {
+  let x = 1;
+
+  for (let i = 0; i < array.length; i++) {
+    if (value === array[i]) {
+      x = 1;
+    } else {
+      x = -1;
+    }
+  }
+  return x;
+}
```

Comment:
this is not right. You should return the index of the value if it is found in the array, else return -1